core/src/filter/resources/jenkins/model/jenkins-version.properties

```diff
@@ -1 +1,2 @@
-version=${build.version}
\ No newline at end of file
+version=${build.version}
+java.min.level=${java.level}
```

Comment:
properties should not start with "`java.`" as that namespace should be considered reserved for properties coming from java itself (to avoid any future blow up when Java introduces a new property)

Reply:
Would "jenkins.java.min.level" work for you?

Reply:
it would work but "min and level" are not really hierachical - so `jenkins.java.minLevel` or `jenkins.java.minVersion` would be preferable in my view.